docs/src/components/pricing/PricingTable.tsx

```diff
@@ -558,6 +558,33 @@ const rowHeaders: Record<string, React.ReactNode> = {
   ),
   'date-picker/simple': <ColumnHead label="Date Picker" />,
   'date-picker/range': <ColumnHead label="Date Range Picker" />,
+
+  // -- charts - components --
+  'charts/line': <ColumnHead label="Line chart" href="/x/react-charts/lines/" />,
+  'charts/bar': <ColumnHead label="Bar chart" href="/x/react-charts/bars/" />,
+  'charts/scatter': <ColumnHead label="Scatter chart" href="/x/react-charts/scatter/" />,
+  'charts/pie': <ColumnHead label="Pie chart" href="/x/react-charts/pie/" />,
+  'charts/sparkline': <ColumnHead label="Sparkline" href="/x/react-charts/sparkline/" />,
+  'charts/gauge': <ColumnHead label="Gauge" href="/x/react-charts/gauge/" />,
+  'charts/treemap': <ColumnHead label="Treemap" href="/x/react-charts/tree-map/" />,
+  'charts/heatmap': <ColumnHead label="Heatmap" href="/x/react-charts/heat-map/" />,
```

Comment:
To match https://mui.com/x/react-charts/heat-map/?

```suggestion
  'charts/heatmap': <ColumnHead label="Heat map" href="/x/react-charts/heat-map/" />,
```